src/gwc/src/main/java/org/geoserver/gwc/GWC.java

```diff
@@ -672,6 +675,29 @@ public final ConveyorTile dispatch(final GetMapRequest request,
             return null;
         }
 
+        if (getConfig().isSecurityEnabled()) {
+            String bboxstr = request.getRawKvp().get("BBOX");
+            String srs = request.getRawKvp().get("SRS");
+            ReferencedEnvelope bbox = null;
+            try {
+                bbox = (ReferencedEnvelope) new BBoxKvpParser().parse(bboxstr);
+            } catch (Exception e) {
+                throw new RuntimeException("Invalid bbox for layer '" + layerName + "': " + bboxstr);
+            }
+            if (srs != null) {
+                try {
+                    bbox = new ReferencedEnvelope(bbox, CRS.decode(srs));
```

Comment:
Hum... will this actually work for wms 1.3 where some crs are flipped, but keep on using the EPSG:XYZW syntax? The WMS request reader checks the version number and changes the srs syntax accordingly in these cases.